reana_job_controller/slurmcern_job_manager.py

```diff
@@ -26,10 +26,12 @@ class SlurmJobManagerCERN(JobManager):
     """Hostname of SLURM head-node used for job management via SSH."""
     SLURM_HEADNODE_PORT = os.getenv("SLURM_CLUSTER_PORT", "22")
     """Port of SLURM head node."""
-    SLURM_PARTITION = os.getenv("SLURM_PARTITION", "batch-short")
+    SLURM_PARTITION = os.getenv("SLURM_PARTITION", "inf-short")
     """Default slurm partition."""
     SLURM_HOME_PATH = os.getenv("SLURM_HOME_PATH", "")
     """Default SLURM home path."""
+    SLURM_TIME = os.getenv("SLURM_TIME", "60")
```

Comment:
Is the limit in minutes or seconds?

Reply:
Minutes, I've added a description about the format in the `config.py` and also a link which points to the official docs